libminifi/src/FlowController.cpp

```diff
@@ -216,9 +215,6 @@ bool FlowController::applyConfiguration(const std::string &source, const std::st
   bool started = false;
   try {
     load(this->root_, true);
-    flow_update_ = true;
-    started = start() == 0;
```

Comment:
Was this supposed to be removed? The code in the conditional branch `if (started) {` below now seems to be dead code.

Reply:
Honestly, I don't remember what was exactly extracted into this PR. I think this was opened to ease the review load on #890, but the boundaries are probably not correctly set. Closing this PR.